.eslintrc.json

```diff
@@ -62,6 +62,15 @@
         ]
       }
     ],
+    "react/no-unknown-property": [
+      2,
+      {
+        "ignore": [
```

Comment:
We use `referrer=noreferrer` in several links and also `mask-type` in svg. `mask-type` is a valid option but it's a known issue for eslint to throw an error. for `referrer` I cannot find the attribute anymore in the known anchor attributes. Should a new task be created to replace those occurrences with 'referrerPolicy' perhaps?

Reply:
Taking a look at this `referrer` is actually incorrect. That should be `rel`. I think most places we use the correct name as well though, but it'd be good to remove the unnecessary extra attribute

Reply:
It used to be an actual attribute for html (referrer) for not sending the referrer header but I think it was years ago. Now the only one I find is `referrerpolicy` ( https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link ).

Maybe change to referrerpolicy or remove alltogether?

Reply:
I opted for removing the `referrer` html attribute from the elements, since it doesn't exist and the `rel="noreferrer"` attribute does the same thing. @hmhealey could you please rereview?